Amplify/Categories/API/ClientBehavior/APICategory+GraphQLBehavior.swift

```diff
@@ -9,6 +9,7 @@ extension APICategory: APICategoryGraphQLBehavior {
     
     // MARK: - Request-based GraphQL operations
 
+    @available(*, deprecated, renamed: "query(request:)")
```

Comment:
These deprecated methods will be removed in another PR, once we get the DataStore implementation updated